diff --git a/node_modules/expo-asset/plugin/build/utils.js b/node_modules/expo-asset/plugin/build/utils.js
index 5b6d8f2..4d0d54e 100644
--- a/node_modules/expo-asset/plugin/build/utils.js
+++ b/node_modules/expo-asset/plugin/build/utils.js
@@ -9,7 +9,7 @@ exports.validateAssets = validateAssets;
 const config_plugins_1 = require("expo/config-plugins");
 const promises_1 = __importDefault(require("fs/promises"));
 const path_1 = __importDefault(require("path"));
-exports.IMAGE_TYPES = ['.png', '.jpg', '.gif'];
+exports.IMAGE_TYPES = ['.png', '.jpg', '.gif', '.webp'];
 exports.FONT_TYPES = ['.otf', '.ttf'];
 exports.MEDIA_TYPES = ['.mp4', '.mp3', '.lottie', '.riv'];
 exports.ACCEPTED_TYPES = ['.json', '.db', ...exports.IMAGE_TYPES, ...exports.MEDIA_TYPES, ...exports.FONT_TYPES];
